### PR TITLE
Disables quick start guide and fixes small typos

### LINKED ISSUE #3888: Broken formatting at habitat.sh
Broken formatting at https://www.habitat.sh/docs/best-practices/#quickstart-guide
Installation section, missed `[` here
`ses on github](https://github.com/habitat-sh/habitat/releases).`

Update:
the same page, seems that this line is not complete:
`How does the package integrate into their`, section `Readme`

Section `Override The Build Phases You Don't Need`
`{: .language-shell}`

**Update**:
page https://www.habitat.sh/docs/best-practices/#quickstart-guide
Section `Override The Build Phases You Don't Need`

**Update**:
page: https://www.habitat.sh/docs/reference/#environment-variables
at the `pkg_description` variable:
`{::comment}` and `{:/comment}`

### PR DESCRIPTION
**Pull request by rsertelon**
Closes #3888

For disabling the quick start, this has been discussed with @eeyun on slack, as written in #3888

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by davidwrede** [comment]
👍

**Comment by tashimi** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/295886071).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/295886071).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/295886071).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/295886071).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by rsertelon** [comment]
Is there any way I can check acceptance myself? or is it eyes only ? ;)

**Comment by tashimi** [comment]
@rsertelon you can check it at https://www.acceptance.habitat.sh/

File: www/source/docs/best-practices.html.md.erb [hunk 1]
```diff
@@ -4,7 +4,6 @@ This chapter covers recommended best practices for runtime and buildtime.
 We focus on best practices for packages that use Habitat Builder for continuous builds.
 
 ## Table of Contents
-  - [Quickstart Guide](#quickstart-guide)
```
Comment (tashimi) [inline www/source/docs/best-practices.html.md.erb]: the link also needs to be removed from the sidebar

File: www/source/partials/docs/_bp-binary-wrapper-packages.html.md.erb [hunk 2]
```diff
@@ -23,7 +23,6 @@ do_install() {
   chmod +x $pkg_prefix/bin/hello_world
 }
 ~~~
-{: .language-shell}
```
Comment (tashimi) [inline www/source/partials/docs/_bp-binary-wrapper-packages.html.md.erb]: i think this looks good!

File: www/source/partials/docs/_bp-planwriting-guide.html.md.erb [hunk 3]
```diff
@@ -106,4 +106,4 @@ All plans should have a README. Items to strongly consider including:
 - Documentation on how to scale the service.
 - Instructions on how to monitor the health of the service at the application layer.
 - Can a user simply call the package as a dependency of their application?
-- How does the package integrate into their 
\ No newline at end of file
+- How does the package integrate into theirs
```
Comment (tashimi) [inline www/source/partials/docs/_bp-planwriting-guide.html.md.erb]: this sentence could be reworded to something like "How does this package integrate into their application?"

File: www/source/partials/docs/_quickstart.html.md.erb [hunk 4]
```diff
@@ -11,7 +11,7 @@ This guide covers how to quickly get started with Habitat
 
 Download the latest release of the Habitat cli tool <code>hab</code>. We are working on supporting tools like homebrew but for now simply visit our [downloads page](https://www.habitat.sh/download/).
 
-Looking for more information or if you'd like to grab the source of a specific release feel free to peruse our releases on github](https://github.com/habitat-sh/habitat/releases).
+Looking for more information or if you'd like to grab the source of a specific release feel free to peruse our releases on [github](https://github.com/habitat-sh/habitat/releases).
```
Comment (tashimi) [inline www/source/partials/docs/_quickstart.html.md.erb]: i'm confused, i thought this was removing the quick start partial - where does this render?

File: www/source/partials/docs/_reference-plan-settings.html.md.erb [hunk 5]
```diff
@@ -189,7 +189,7 @@ In this example, the corresponding default.toml file would have the following ke
   ~~~
 
 ### pkg_description
-**Required** for [core](https://github.com/habitat-sh/core-plans) plans, optional otherwise. A short description of the package. It can be a simple string, or you can create a multi-line description using markdown to provide a rich description of your package. {::comment} This description will be displayed on the Web app when users search for or browse to your package. {:/comment}
+**Required** for [core](https://github.com/habitat-sh/core-plans) plans, optional otherwise. A short description of the package. It can be a simple string, or you can create a multi-line description using markdown to provide a rich description of your package. **This description will be displayed on the Web app when users search for or browse to your package.**
```
Comment (tashimi) [inline www/source/partials/docs/_reference-plan-settings.html.md.erb]: looks good!

File: www/source/partials/docs/_quickstart.html.md.erb (hunk 4 above)
Comment (rsertelon) [reply www/source/partials/docs/_quickstart.html.md.erb]: It is more a deactivation until it's ready.

File: www/source/docs/best-practices.html.md.erb (hunk 1 above)
Comment (rsertelon) [reply www/source/docs/best-practices.html.md.erb]: @tashimi I don't know find where sidebar contents are written :/ I can find the sidebar layout, but not the content...

File: www/source/partials/docs/_bp-planwriting-guide.html.md.erb (hunk 3 above)
Comment (rsertelon) [reply www/source/partials/docs/_bp-planwriting-guide.html.md.erb]: Done :)

File: www/source/partials/docs/_quickstart.html.md.erb (hunk 4 above)
Comment (rsertelon) [reply www/source/partials/docs/_quickstart.html.md.erb]: @eeyun can you confirm this is still a thing? :)

File: www/source/docs/best-practices.html.md.erb (hunk 1 above)
Comment (rsertelon) [reply www/source/docs/best-practices.html.md.erb]: I've found it. Done ;)

**Comment by tashimi** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/296519223).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/296519223).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/296519223).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by tashimi** [comment]
tenor-91176114

Looks good! 

@thesentinels approve

At some point we will want to delete or finish the quickstart partial - @davidwrede do you think we need it, or do we have the same content covered in the tutorials?

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/296525567).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/296525567).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/296525567).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/296525567).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by davidwrede** [comment]
@tashimi Feels like this quickstart content is already covered in our existing tutorials.

**Comment by tashimi** [comment]
@davidwrede sounds good to me!

**Pull request merged**